test/src/test/java/hudson/cli/DisconnectNodeCommandTest.java

```diff
@@ -264,6 +265,8 @@ public void disconnectNodeManyShouldSucceedEvenANodeIsSpecifiedTwice() throws Ex
         assertThat(((OfflineCause.ByCLI) slave2.toComputer().getOfflineCause()).message, equalTo("aCause"));
     }
 
+    @Ignore("TODO currently failing")
```

Comment:
The test has not been enabled since it was first written in 2016 as part of 6feaea8647fe328eb47ea764764897cdb212f3fb .  Could we delete the test rather than add a TODO to diagnose a test that has never been executed since the test class was created?

Reply:
Not opposed though it is likely in the scope for another PR.

Reply:
Sure, I deleted the test.